users/models.py

```diff
@@ -31,11 +30,21 @@ def _create_user(
         Creates and saves a user with the given email and password.
         Returns created user.
         """
+
         if not email:
             raise CustomValidationError(
                 CustomValidationErrorText.VALIDATION_ERROR_EMAIL_MESSAGE,
                 ErrorCode.CREATE_USER_EMAIL_MISSING,
             )
+        else:
+            domain = email.split('@')[1]
```

Comment:
What if someone would type `xxx@codepoets.it@gmail.com`? This is still valid, but it shouldn't

Reply:
If we take into consideration only this part of code, it is valid. But built-in Django validators would not pass it. Even if I would handle every exception, in first way Django would throw something like:  `ValidationError: {'email': ['Enter a valid email address.']}`.

Reply:
I add verification for that, so it's **Done**